### PR TITLE
Enums

### PR DESCRIPTION
**Pull request by MarzaElise**
Made enums for some guild attributes since a literal int wont really mean anything. Didnt change any documentation cos i suck at them. I still feel like class names could be changed, but i dont know so yea... :)

### CODE REVIEW

File: acord/models/guild.py [hunk 1]
```diff
@@ -41,7 +49,7 @@ class Guild(pydantic.BaseModel, Hashable):
 
     channels: List[Any]  # This gets sorted out with validators
     """ All channels in the guild """
-    default_message_notifications: Literal[0, 1]
+    default_message_notifications: GuildMessageNotification
```
Comment (Seniatical) [inline acord/models/guild.py:52]: :class:\`GuildMessageNotification\`

File: acord/models/guild.py [hunk 2]
```diff
@@ -56,7 +64,7 @@ class Guild(pydantic.BaseModel, Hashable):
 
     emojis: List[Emoji]
     """ List of emojis in guild """
-    explicit_content_filter: int
+    explicit_content_filter: ExplicitContentFilterLevel
```
Comment (Seniatical) [inline acord/models/guild.py:67]: :class:\`GuildMessageNotification\`

File: acord/models/guild.py [hunk 3]
```diff
@@ -94,7 +102,7 @@ class Guild(pydantic.BaseModel, Hashable):
     members: Dict[Snowflake, Member]
     """ Mapping of all members in guild """ 
 
-    mfa_level: int
+    mfa_level: MFALevel
```
Comment (Seniatical) [inline acord/models/guild.py:105]: :class:\`MFALevel\`

File: acord/models/guild.py [hunk 4]
```diff
@@ -103,7 +111,7 @@ class Guild(pydantic.BaseModel, Hashable):
 
     nsfw: bool
     """ Whether the guild is marked as NSFW """
-    nsfw_level: int
+    nsfw_level: NSFWLevel
```
Comment (Seniatical) [inline acord/models/guild.py:114]: :class:\`NSFWLevel\`

File: acord/models/guild.py [hunk 5]
```diff
@@ -121,7 +129,7 @@ class Guild(pydantic.BaseModel, Hashable):
     """ Whether Boosts progress bar is enabled """
     premium_subscription_count: int
     """ Number of guild boosts """
-    premium_tier: int
+    premium_tier: PremiumTierLevel
```
Comment (Seniatical) [inline acord/models/guild.py:132]: :class:\`PremiumTierLevel\`

File: acord/models/guild.py [hunk 6]
```diff
@@ -159,7 +167,7 @@ class Guild(pydantic.BaseModel, Hashable):
     vanity_url_code: Optional[str]
     """ the vanity url code for the guild """
 
-    verification_level: int
+    verification_level: VerificationLevel
```
Comment (Seniatical) [inline acord/models/guild.py:170]: :class:\`VerificationLevel\`

**Review by Seniatical — CHANGES_REQUESTED** [review]
Fix docstrings, possibly shift to `acord.bases`. And create new folder called enums, which contains GuildEnums

File: acord/models/guild.py (hunk 1 above)
Comment (MarzaElise) [reply acord/models/guild.py:52]: image
 is this what you mean?

**Comment by MarzaElise** [comment]
> Fix docstrings, possibly shift to `acord.bases`. And create new folder called enums, which contains GuildEnums

so like `acord/bases/GuildEnums/xxxxx.py?`

**Comment by Seniatical** [comment]
possibly acord/bases/enums/guild.py

File: acord/models/guild.py (hunk 1 above)
Comment (Seniatical) [reply acord/models/guild.py:52]: remove the double \`, you only need single ones

Comment (MarzaElise) [reply acord/models/guild.py:52]: okithen

File: acord/models/guild.py (hunk 2 above)
Comment (MarzaElise) [reply acord/models/guild.py:71]: Done in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c

File: acord/models/guild.py (hunk 4 above)
Comment (MarzaElise) [reply acord/models/guild.py:115]: Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c

File: acord/models/guild.py (hunk 5 above)
Comment (MarzaElise) [reply acord/models/guild.py:130]: Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c

File: acord/models/guild.py (hunk 6 above)
Comment (MarzaElise) [reply acord/models/guild.py:165]: Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c

File: acord/models/guild.py (hunk 3 above)
Comment (MarzaElise) [reply acord/models/guild.py:107]: Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c

File: acord/models/guild.py (hunk 1 above)
Comment (MarzaElise) [reply acord/models/guild.py:57]: Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c

**Comment by Seniatical** [comment]
having a look

**Review by Seniatical — APPROVED** [review]
Looking good, will need to import from `acord/bases/__init__.py`

**Comment by Seniatical** [comment]
import needed so the docs can find the classes 💀

**Comment by MarzaElise** [comment]
i absolutely forgot about imports... i need a proper linter

**Comment by Seniatical** [comment]
🥳 merged

**Pull request merged**